### ISSUE TITLE
Some issues from Alyssa Demo Runthrough

### ISSUE THREAD

**Issue by johnarban** [cosmicds/hubbleds#785]
- Make sure font sizes for tables and labels, etc are all well sized 
- On `you_age1c` show students confidence interval on the plot. 
- when there is no wrong answers, we should use white. the orange still seems wrong. 

- Add some "Classmate 1, 2,...." and "Your Meas"
- "My age" should be relabled (it feels like 
- Professional data viewers - turn on the best fit line automatically
- Warning boxes should be styled the same as incorrect multiple choice answers
- maybe for the the prodata "who do i believe" should they be able to change their mind
- make sure stage 4 and 5 load well
